pkgs/servers/tautulli/default.nix

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Please cleanup the unused inputs.

Reply:
Thanks for the feedback!
- I removed the unused inputs
- tested the package by deploying it on my server
- > worked without any problems